gobblin-runtime/src/main/java/org/apache/gobblin/runtime/SafeDatasetCommit.java

```diff
@@ -210,6 +210,7 @@ private void maySubmitLineageEvent(JobState.DatasetState datasetState) {
       }
     }
     if (states.size() == 0) {
+      log.info("Will not submit lineage events");
```

Comment:
maybe change it to include the reason why? 
"will not submit lineage events due to empty tasks"

Reply:
+1